arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+const MAXIMIZE = "maximize";
```

Comment:
these could arguably be moved to `constants.ts`

Reply:
This seems odd, since the constant is only really used locally.  If we were reusing it across multiple components, that would make more sense.